source/inputCore.py

```diff
@@ -492,7 +492,12 @@ def executeGesture(self, gesture):
 			queueHandler.queueFunction(queueHandler.eventQueue, speech.speakMessage, gesture.displayName)
 
 		gesture.reportExtra()
-
+		
+		# Clear memorized last script to avoid getLastScriptRepeatCount detect a repeat
+		# in case an unbound gesture is executed between two identical bound gestures.
+		if not script:
+			scriptHandler.clearLastScript()

```

Comment:
> This needs to be queued to the main thread.

Done. Thanks.

> I would also consider moving this code into an else block directly under the call to queueScript at the end of this function.


As you suggested, I have moved the call to clearLastScript at the end of the function. Actually, clearing last script variables before handling intercepted command Script case would maybe have caused wrong setting of the variable keeping track of the last executed script.

I have also moved the `raise NoInputGestureAction` in the else branch since the call to clearLastScript and the raise statement belong to the same execution branch.